commit 15d0a559f97219956007e300f0334b20412df1f9 (refs/changes/13/3113/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-12-14T12:01:57-08:00 (1 year, 10 months ago)
    
    DOCKER-1134 nfs volumes tests run even though nfs volumes are disabled

diff --git a/docs/api/features/volumes.md b/docs/api/features/volumes.md
index aa8f3a5..f15e7cf 100644
--- a/docs/api/features/volumes.md
+++ b/docs/api/features/volumes.md
@@ -28,8 +28,8 @@ line from the head node:
 
 ```
 sdcadm post-setup volapi
-sdcadm experimental docker-nfs-volumes
-sdcadm experimental docker-nfs-volumes-automount
+sdcadm experimental nfs-volumes docker
+sdcadm experimental nfs-volumes docker-automount
 ```
 
 This command will create a new core zone that runs the VOLAPI service, which
@@ -48,7 +48,7 @@ SAPI to disable support for NFS shared volumes by running the following command
 line:
 
 ```
-sdcadm experimental docker-nfs-volumes -d
+sdcadm experimental nfs-volumes docker -d
 ```
 
 After disabling this setting, running `docker volume` commands will result in an
diff --git a/test/integration/api-mount-nfs-shared-volume-as-docker-compose.test.js b/test/integration/api-mount-nfs-shared-volume-as-docker-compose.test.js
index e39a3e7..22fa3f1 100644
--- a/test/integration/api-mount-nfs-shared-volume-as-docker-compose.test.js
+++ b/test/integration/api-mount-nfs-shared-volume-as-docker-compose.test.js
@@ -20,15 +20,13 @@
  */
 
 
-var jsprim = require('jsprim');
-
 var common = require('../lib/common');
 var dockerTestHelpers = require('./helpers');
 var mod_testVolumes = require('../lib/volumes');
 var volumesApi = require('../lib/volumes-api');
 
 var test = mod_testVolumes.createTestFunc({
-    checkTritonSupportsNfs: true
+    checkTritonSupportsNfsVols: true
 });
 
 var ALICE_ACCOUNT;
diff --git a/test/integration/api-rename-used-nfs-shared-volume.test.js b/test/integration/api-rename-used-nfs-shared-volume.test.js
index ceac9f6..6f28e22 100644
--- a/test/integration/api-rename-used-nfs-shared-volume.test.js
+++ b/test/integration/api-rename-used-nfs-shared-volume.test.js
@@ -21,7 +21,7 @@ var mod_testVolumes = require('../lib/volumes');
 var volumesApi = require('../lib/volumes-api');
 
 var test = mod_testVolumes.createTestFunc({
-    checkTritonSupportsNfs: true
+    checkTritonSupportsNfsVols: true
 });
 
 var ALICE_ACCOUNT;
